src/main/java/com/kdt_y_be_toy_project2/domain/trip/controller/TripController.java

```diff
@@ -0,0 +1,53 @@
+package com.kdt_y_be_toy_project2.domain.trip.controller;
+
+import com.kdt_y_be_toy_project2.domain.trip.dto.*;
+import com.kdt_y_be_toy_project2.domain.trip.service.TripService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/v1/trips")
+@RequiredArgsConstructor
+public class TripController {
+    private final TripService tripService;
+
+    @GetMapping
+    public ResponseEntity<List<GetTripResponse>> getAllTrips() {
+        return ResponseEntity.status(HttpStatus.OK)
+                .contentType(MediaType.APPLICATION_JSON)
+                .body(tripService.getAllTrips());
+    }
+
+    @GetMapping("/{trip_id}")
+    public ResponseEntity<GetTripResponse> getTripById(
+            @PathVariable(name = "trip_id") final Long tripId
+    ) {
+        return ResponseEntity.status(HttpStatus.OK)
+                .contentType(MediaType.APPLICATION_JSON)
+                .body(tripService.getTripById(tripId));
+    }
```

Comment:
요건 팁이긴 한데 단순하게 `HttpStatus` 가 OK 이고 `contentType`이 JSON 이라면 굳이 ResponseEntity에 담아 주지 않아도 됩니다.

```suggestion
    @GetMapping("/{trip_id}")
    public GetTripResponse getTripById(
            @PathVariable(name = "trip_id") final Long tripId
    ) {
        return tripService.getTripById(tripId);
    }
```

Reply:
여정팀 prefix 맞춰서 진행하겠습니다.